src/Cargo.lock

```diff
@@ -255,16 +255,16 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "clap"
-version = "2.25.1"
+version = "2.26.0"
```

Comment:
Just edit the `Cargo.lock` by hand to make the removals. Conversely, revert the unrelated changes. Cargo should then not regenerate the lock file.